npm/private/npm_import.bzl

```diff
@@ -619,7 +619,10 @@ def _impl_links(rctx):
             msg = "lifecycle_hooks_env contains invalid key value pair '%s', required '=' separator not found" % env
             fail(msg)
 
-    lifecycle_hooks_execution_requirements = {}
+    lifecycle_hooks_execution_requirements = {
+        # Workaround engflow actions having no DNS despite requires-network tag
+        "no-remote-exec": "1",
```

Comment:
you can set the lifecycle hooks for the specific offending packages using `rctx.attr.lifecycle_hooks_execution_requirements` in `npm_translate_lock`

Reply:
hard to know which ones they are though. the figma package suggests that node-gyp always talks to nodejs.org

Reply:
okay I'll just repeat until I identify them all, for this PR at least